embeddings/pipeline/lightning_hps_pipeline.py

```diff
@@ -66,17 +70,23 @@ class OptimizedLightingPipeline(
         str,
         datasets.DatasetDict,
         datasets.DatasetDict,
+        Dict[str, nptyping.NDArray[Any]],
+        Dict[str, Any],
     ],
     AbstractHuggingFaceOptimizedPipeline[ConfigSpace],
     _OptimizedLightingPipelineBase[ConfigSpace],
     ABC,
     Generic[ConfigSpace, LightningMetadata],
 ):
-    def _get_evaluation_metadata(self, parameters: SampledParameters) -> LightningMetadata:
+    def _get_evaluation_metadata(
+        self, parameters: SampledParameters, trial_name: str = "", **kwargs: Any
+    ) -> LightningMetadata:
         metadata = self._get_metadata(parameters)
         metadata["predict_subset"] = LightingDataModuleSubset.VALIDATION
         metadata["dataset_name_or_path"] = str(self.dataset_path)
-        metadata["output_path"] = self.tmp_model_output_dir.name
+        output_path = Path(self.tmp_model_output_dir.name).joinpath(trial_name)
```

Comment:
why Path is created in that function? It should return only metadata

Reply:
It is just a subdir for single trail. Do you think it should be separated anyway?

Reply:
I think it should be seperated, as I wouldn't suspect creating directories while getting Metadata.